src/pages/ProvideLiquidity/index.tsx

```diff
@@ -75,10 +75,12 @@ export default function ProvideLiquidity() {
         <CardSection>
           <AutoColumn gap="md">
             <RowBetween>
-              <TYPE.white fontWeight={600}>Providing liquidity in Revo is still in Beta</TYPE.white>
+              <TYPE.white fontWeight={600}>Providing liquidity on Revo</TYPE.white>
```

Comment:
was wondering, can we remove this card and put this information in the blue box? then this tab can be consistent with the "zap" tab, with the tab information in the blue box